src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -320,6 +324,9 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
   // TODO Add support for uploaded Project Area shapefiles
   createScenario(): void {
     this.generatingScenario = true;
+    this.planService.createScenario(this.formValueToProjectConfig()).subscribe();
+    const planId = this.plan$.getValue()?.id;
```

Comment:
This should probably go inside the `subscribe()`  no ? Like we want to get the id after we created the scenario, and redirect after we know it saved correctly.
Also, to note we should probably handle errors as well